### PR TITLE
Support update combining doc and build fixes

### PR DESCRIPTION
**Pull request by CollinAvidano**
Still needs work on docs I just wanted to make sure there is nothing you all see that would be breaking or super annoying to deal with point is to make it easier for new members

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,15 +60,22 @@ $ cd robocup-firmware
 $ ./util/<SYSTEM>-setup
 ```
 
-3) Build the project for the desired target. The `control` target is the firmware for the MBED. The `fpga` target is for the FPGA to be uploaded to the MBED. The `kicker` target is for the kicker MCU to be uploaded to the MBED. The `base` target is for the base station firmware to be uploaded to the base station MBED.
+This will install conan for you if you have not previously installed it. If you respond no to the setup script see the note under Setting up Conan for RoboJackets Firmware in [Getting Started](doc/GettingStarted.md) before continuing.
+
+3) Build the project for the desired target. The `robot` target is the firmware for the MTrain. The `kicker` target is for the kicker MCU to be uploaded to the MTrain. The `clean` target deletes the build directories for both robot and kicker firmware.
```
Comment (JNeiger) [inline README.md]: This doesn't actual upload stuff to mtrain and there's extra stuff you have to do for the kicker. I would either describe those here or make a note that there's more steps involved

File: doc/Conan.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+## Binary Management
+Conan also is capable of managing binaries targeted at specific platforms within these binaries. The settings for building for a specific platform are described in a conan profile which are stored in ~/.conan/profiles/ . For robocup-firmware you will be using the will be using the armv7hf profile to build for the mtrain platform.
+
+
+## Creating a local conan package:
+For those who are working on both mtrain and robocup firmware you may want to have seperate testing version of the mtrain package to do so would make a package with a different reference (which makes the package into a different folder in your ~/.conan/data/ folder) and then change the package dependencies in robocup-firmware to match the new package reference.
+
+The below is used to create a local version of the conan package:
+```
+conan create . collin/testing -pr armv7hf
```
Comment (JNeiger) [inline doc/Conan.md]: There's usually some normal way to say "put your name here" in commands, not quite sure

**Comment by JNeiger** [comment]
I need to finish up writing the docs for the generic control doc. Totally forgot about that. Don't block this on that though, it's not that important.

File: README.md (hunk 1 above)
Comment (JNeiger) [reply README.md]: For kicker and the fpga, you have to run the convert.py file which outputs a `*_bin.h` file for the binarys. You throw those in the `robot/lib/Inc/device-bins/` and reupload to mtrain. `make control-upload` is the mtrain upload iirc.

Comment (CollinAvidano) [reply README.md]: I didnt write this part it was just left over I just changed it to say conan is a thing and changed the names to the new build targets.I will say that robot doesnt upload either this section really doesnt explain how to upload. I explain how to manually upload in the FlashingMtrain.md document but I should probably link that here as well as either make one for kicker (I like the seperated single topic md tutorials if you cant tell) or add it to the flashing mtrain one and rename it FlashingHardware.md

File: doc/Conan.md (hunk 2 above)
Comment (CollinAvidano) [reply doc/Conan.md]: Usually you would do <yourname> . That or [yourname] or something of the sort. I kinda mentioned at the top that [ ] means optional but didnt really continue to put it in every case it was true. I am always afraid of doing this as many 2200 students will put their names in angle brackets and then ask they the command we gave them failed. I may just change it to yourname/testing

File: README.md [hunk 3]
```diff
@@ -12,32 +12,45 @@ The official [RoboCup site](http://robocupssl.cpe.ku.ac.th/) has more informatio
...
 
-Compiled programs and some configuration files are stored here.
+### kicker/
+The kicker folder contains the firmware code for the different targets that are compiled for the kicker board.
+Details on the architecture of the kicker firmware code can be found [here](doc/Kicker.md)
 
+### kicker/build/
+Compiled binaries output from the build system for the ATtiny on the kicker board get stored here in the bin/ directory as well as other build results.
+
+### /fpga
```
[the remark this answers is not in the document]
Comment (CollinAvidano) [reply README.md]: Whoops. Thats a typo.

File: doc/Conan.md (hunk 2 above)
Comment (JNeiger) [reply doc/Conan.md]: Ooo, yeah, maybe not then. This was definitely on the bottom of things that might cause problems. Do whatever works in 2200

File: README.md (hunk 1 above)
Comment (JNeiger) [reply README.md]: That should be fine then. I just wanted to make sure it was very clear that more needs to happen before you can restart the robot and hope the code works.

**Pull request closed without merging**